CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## v2.11.0-rc.0 / 2019-07-03
+
+* [CHANGE] Remove `max_retries` from queue_config (it has been unused since rewriting remote-write to utilize the write-ahead-log). #5649
+* [CHANGE] The meta file `BlockStats` no longer holds size information. This is now dynamically calculated and kept in memory. It also includes the meta file size which was not included before. tsdb#637
+* [CHANGE] Create new clean segment when starting the WAL. tsdb#608
```

Comment:
This is not a breaking change, I'm not sure it's even worth mentioning - ENHANCEMENT a most.